contract-message-predicate/src/transaction_utils.sw

```diff
@@ -0,0 +1,118 @@
+library transaction_utils;
+
+use std::constants::ZERO_B256;
+use std::mem::read;
+
+// TODO: replace GTF consts with direct references to tx.sw, inputs.sw, and outputs.sw from std lib
```

Comment:
I assume that all of this will be in the `std`? I guess that direct usage of the opcode will be replaced by some function in the `intrinsics` section of the `std` too.

Reply:
Possibly. At the very least I expect them to be exposed in `tx.sw`, `inputs.sw`, and `outputs.sw`. See https://github.com/FuelLabs/sway/pull/2707#issuecomment-1266255843